src/app/components/layout/header/header.component.html

```diff
@@ -20,6 +20,14 @@
     />
 
     <div class="flex">
+      <span
```

Comment:
use <a tag so the text has a pointer and we can open it in a new tab if we want by right clicking